ovirt_imageio/_internal/http.py

```diff
@@ -72,6 +72,11 @@ class Server(socketserver.ThreadingMixIn,
     """
     daemon_threads = True
 
+    # Override socketserver.TCPServer default (5) to allow larger number
+    # of concurrent connections. If larger number of connections are
+    # made, some connections will fail with EAGAIN.
+    request_queue_size = 40
```

Comment:
Would be useful to make it configurable instead of hardcoding another value? If we want to stick with max 10 concurrent image download, it would be good to add a comment about it here.

Reply:
Configuration is a problem since this server is not designed to support many concurrent users. The hard coded limit makes it hard to abuse it.

But a comment about supporting 10 concurrent transfers is good idea.